django/db/migrations/state.py

```diff
@@ -201,7 +201,7 @@ def register_model(self, app_label, model):
             self.app_configs[app_label] = AppConfigStub(app_label)
             self.app_configs[app_label].models = OrderedDict()
         self.app_configs[app_label].models[model._meta.model_name] = model
-        self.clear_cache()
+        self.clear_cache(True)
```

Comment:
Agreed - all other uses have explicitly called `expire_models_cache=True`. However, this is now a moot point since we can remove `expire_models_cache`.